testnets/dymensionplaygroundtestnet/assetlist.json

```diff
@@ -0,0 +1,164 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "dymensionplaygroundtestnet",
+  "assets": [
+    {
+      "description": "The native governance and staking token of the Dymension Hub",
+      "denom_units": [
+        {
+          "denom": "adym",
+          "exponent": 0
+        },
+        {
+          "denom": "dym",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "adym",
+      "name": "Dymension",
+      "display": "dym",
+      "symbol": "DYM",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/dymensionplaygroundtestnet/images/dymension-logo.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/dymensionplaygroundtestnet/images/dymension-logo.svg"
+      },
+      "coingecko_id": "dymension",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/dymensionplaygroundtestnet/images/dymension-logo.png",
```

Comment:
These images aren't identical...
And there already exists dymension logos in the registry, so we don't need duplicates.